python/examples/core/transforms.py

```diff
@@ -262,15 +246,6 @@ class Generalize(Transform):
   def __init__(self, fun_name: str, op_name: str, **kwargs):
     self.fun_name = fun_name
     self.op_name = op_name
-    self._parse_variables_in_kwargs(kwargs)
-    interchange_str = _get_size_list_as_str(name='iterator-interchange',
```

Comment:
is _get_size_list_as_str dead code as well?

Reply:
Not yet, we still have uses in Fuse and in LinalgExtTile but both are arguably broken.
Will cleanup in a later incantation as I bring up LinalgExtTile